src/theme/Checkbox.ts

```diff
@@ -3,21 +3,36 @@ import { PartsStyleFunction } from "@chakra-ui/theme-tools"
 
 const baseStyle: PartsStyleFunction<typeof checkboxAnatomy> = () => {
   return {
+    label: {
+      display: "flex",
+      alignItems: "center",
+      ml: 3,
+      fontWeight: "medium",
+    },
     control: {
+      outline: "3px solid",
+      outlineColor: "transparent",
+      p: 2,
+      borderRadius: "md",
       _checked: {
-        borderColor: "brand.500",
+        borderColor: "hsl(182, 100%, 70%)",
         _active: {
-          borderColor: "brand.500",
+          borderColor: "hsl(182, 100%, 70%)",
         },
-        backgroundColor: "brand.500",
+        bg: "hsl(182, 100%, 70%)",
```

Comment:
Isn't it the same color as `#66F9FF`, which is `branch.100` in our color theme?

If yes, then let's change all the occurences of `hsl(182, 100%, 70%)` in our app to `brand.100`.

Reply:
Ref commit: ad674a24